free-programming-books.md

```diff
@@ -518,6 +518,7 @@
 * [Real-World Maintainable Software](http://www.oreilly.com/programming/free/real-world-maintainable-software.csp) *(Just fill the fields with any values)*
 * [Record-Playback Test Automation: Sahi & Selenium IDE: Critical Evaluation of Record-Playback Automation Tools](https://leanpub.com/manualToAutomatedWithSeleniumIDEAndSahi) - Shashikant Jagtap
 * [Security Engineering](http://www.cl.cam.ac.uk/~rja14/book.html)
+* [Selenium Tutorial for Automation Testing](http://www.techbeamers.com/selenium-webdriver-tutorial)
```

Comment:
Did you consider putting this instead in https://github.com/EbookFoundation/free-programming-books/blob/master/free-programming-interactive-tutorials-en.md ?

Reply:
Thanks for the feedback!

And actually, I'm soon going to give an option to download these Python and Selenium tutorials as free pdf e-books. So, please suggest if it is just okay to add a single reference under the meta-list section of free-programming-books.md.

Reply:
My preference is one listing per ebook/tutorial. Put them in the tutorial list until you have the free pdf, then move them to ebooks. Go ahead and add a meta-list entry. Consider adding the author name 😄 .
But please look at the lists, decide where your tutorials fit best, and let us know you've thought about it. You're in the best position to decide (no matter my preference), and we want to encourage people like techbeamers.

Reply:
Great. After adding the meta-list entry, I've placed the tutorials inside the free-programming-interactive-tutorials-en.md under appropriate sections. Thanks.